.github/workflows/nightly.yml

```diff
@@ -37,7 +37,11 @@ jobs:
         run: cargo build -p xtask
 
       - name: Upgrade Cairo to latest main commit
-        run: cargo xtask set-cairo-version --rev $(git ls-remote --refs "https://github.com/starkware-libs/cairo" main | awk '{print $1}')
+        run: cargo xtask set-dep-version cairo --rev $(git ls-remote --refs "https://github.com/starkware-libs/cairo" main | awk '{print $1}')
+
+      - name: Upgrade CairoLS to latest main commit
+        #        run: cargo xtask set-dep-version cairols --rev $(git ls-remote --refs "https://github.com/software-mansion/cairols" main | awk '{print $1}')
+        run: cargo xtask set-dep-version cairols --rev 94f1543b43caa6355bf492f80cdf46b2ae2bf854
```

Comment:
this is temporary so that I could cut nightly build. I will drop this commit afterwards